lua/orgmode/config/init.lua

```diff
@@ -259,6 +261,35 @@ function Config:get_todo_keywords()
   return types
 end
 
+--- Get a list of priority values based on user configuration of the low/high values
+---
+---@return table<string> priority values in ascending order
+function Config:get_priority_values()
```

Comment:
_todo_: This is a minor duplicate of the code for the priority state stuff, so not sure where the right line is between de-duplicating code versus favoring "simplicity".